CHANGELOG.txt

```diff
@@ -28,6 +28,7 @@ Changelog
  * Fix: The ModelAdmin module can now work without the wagtailimages and wagtaildocs apps installed (Andy Babic)
  * Fix: Cloudflare error handling now handles non-string error responses correctly (hdnpl)
  * Fix: Search indexing now uses a defined query ordering to prevent objects from being skipped (Christian Peters)
+ * Issue 317: Restrict access to the wagtailadmin.pages.view_draft view to Edit / Publish permission. (Kees Hink)
```

Comment:
I also heard that editing the changelog in PRs is a source of headaches, because of merge conflicts. I'll just leave it out for now.